_sass/helix/_tables.scss

```diff
@@ -0,0 +1,66 @@
+/* ==========================================================================
+   Tables
+   ========================================================================== */
+/**
+ * Remove most spacing between table cells.
+ */
+table {
+  border-collapse: collapse;
+  border-spacing: 0; 
+  border:1px solid #e0e0e0;
+  th {
+  	padding: 1.25em;
+  	border-bottom: 2px solid #0C7C84;
+  	text-align: left;
+  	.1-4 {
```

Comment:
These classes starting with numbers will break.  What effect are you going for with these classes?